packages/headless/src/api/commerce/product-listings/query-listing.http

```diff
@@ -0,0 +1,47 @@
+POST http://localhost:8100/rest/organizations/commercestore/properties/propertyid/commerce/v2/listing
+Content-Type: application/json
+Authorization: Bearer x1fafab91-152b-4c2e-9f16-681bc5113722
```

Comment:
Is it problematic to have this key in the clear on the internet ?

Reply:
it's an invalid access token, so no. Removed the file since it was included because I forgot to clean up after myself 😇